lib/pagerduty/http_transport.rb

```diff
@@ -35,6 +35,6 @@ def http
   end
 
   def transported?(response)
-    response.kind_of? Net::HTTPSuccess or response.kind_of? Net::HTTPRedirection
+    response.is_a?(Net::HTTPSuccess) || response.is_a?(Net::HTTPRedirection)
```

Comment:
I'm not fussed with `kind_of?` vs `is_a?`. Is Rubocop enforcing this?

:+1: to changes on this line in any case.